.drone.star

```diff
@@ -74,29 +74,83 @@ config = {
             "suites": [
                 "apiArchiver",
                 "apiContract",
-                "apiGraph",
-                "apiGraphUserGroup",
-                "apiSpaces",
-                "apiSpacesShares",
                 "apiCors",
                 "apiAsyncUpload",
                 "apiDownloads",
-                "apiReshare",
-                "apiSpacesDavOperation",
                 "apiDepthInfinity",
                 "apiLocks",
+                "apiActivities",
+            ],
+            "skip": False,
+        },
+        "apiGraph": {
+            "suites": [
+                "apiGraph",
+            ],
+            "skip": False,
+            "withRemotePhp": [False],
+            "skipRemotePhpTest": True,
+        },
+        "apiGraphUserGroup": {
+            "suites": [
+                "apiGraphUserGroup",
+            ],
+            "skip": False,
+            "withRemotePhp": [False],
+            "skipRemotePhpTest": True,
+        },
+        "apiSpaces": {
+            "suites": [
+                "apiSpaces",
+            ],
+            "skip": False,
+        },
+        "apiSpacesShares": {
+            "suites": [
+                "apiSpacesShares",
+            ],
+            "skip": False,
+        },
+        "apiSpacesDavOperation": {
+            "suites": [
+                "apiSpacesDavOperation",
+            ],
+            "skip": False,
+        },
+        "apiSearch1": {
+            "suites": [
                 "apiSearch1",
+            ],
+            "skip": False,
+        },
+        "apiSearch2": {
+            "suites": [
                 "apiSearch2",
+            ],
+            "skip": False,
+        },
+        "apiSharingNg": {
+            "suites": [
+                "apiReshare",
                 "apiSharingNg1",
                 "apiSharingNg2",
+            ],
+            "skip": False,
+        },
+        "apiSharingNgShareInvt": {
```

Comment:
Updated the name and made it shorter, since to differentiate the pipelines to run without remote.php, the pipeline is named as `API-Tests-apiSharingNgShareInvitation-without-remote.php` which makes the length of the pipeline greater than 50 characters.
P.S. Suggestions for better pipeline names are welcome.